### DISCUSSION TITLE
[rejected] Allegro One - brak przykładów i dokumentacji

### DISCUSSION THREAD

**Discussion by haloween** [allegro/allegro-api#5575] [Suggestions - rejected]
Witam,

opiekunowie ze sprzedaży cisną wszystkich o korzystanie z waszych paczkomatów ale dokumentacja API niestety "ssie". 
Przydały by się jakieś przykłady, jak wykonać całość flow. 

Pozdrawiam

**Comment by MartaNowaczyk** [comment]
Dzięki, jeśli będziemy coś zmieniać w naszych [poradnikach](https://developer.allegro.pl/wza/) weźmiemy to pod uwagę.

**Reply by haloween** [reply]
To może napiszcie do opiekunów klientów że nie macie zrobionej dokumentacji i sprzedawcom się ciężko integrować.
Bo ja nie jestem w stanie zgadnąć z dokumentacji co autor miał na myśli, a jakiś flowcharcik by ułatwił życie.

**Reply by MartaNowaczyk** [reply]
Opis i dokumentacja są dostępne. Listę punktów Allegro pobierzesz za pomocą [GET /order/carriers/ALLEGRO/points](https://developer.allegro.pl/documentation/#operation/getAllegroPickupDropOffPointsGET). Spójrz także do [naszego poradnika](https://developer.allegro.pl/wza/#jak-pobra%c4%87-list%c4%99-punkt%c3%b3w-allegro), gdzie opisaliśmy ten proces. Raczej chodziło mi bardziej o kosmetyczne zmiany, albo zmianę sposobu przekazania informacji, ale same informacje są i integratorzy z nich korzystają.
